slo-monitor/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+- gmarek
+- wojtek-t
+reviewers:
+- gmarek
+- wojtek-t
+
```

Comment:
It's not covered yet, but no harm in having it here if at some point we'll enable tooling for whole k8s.io org.